.circleci/php-5.5_web.txt

```diff
@@ -0,0 +1,14 @@
+test-metrics
+cakephp-28-suite
+codeigniter-22-test
+laravel-42-suite
+lumen-52-suite
```

Comment:
It looks like [Slim 3 supports PHP 5.5](http://www.slimframework.com/docs/v3/start/installation.html) so it might be good to add that to the list.

Reply:
Are you okay if I handle this in the follow-up PR?

Reply:
Yep! :)